### DISCUSSION TITLE
How to deploy Directus 9 to qovery.com

### DISCUSSION THREAD

**Discussion by rashidpathiyil** [directus/directus#4605] [Archive]
How to deploy Directus 9 to qovery.com with Postgres + Qovery Storage

**Comment by evoxmusic** [comment]
Hi @rashidpathiyil, I am the co-founder of Qovery and I guess it's possible to deploy Directus as they support Docker (https://docs.directus.io/guides/installation/docker/). I will be happy to help and see how we can officially support Directus by providing a one-click template deployment in Qovery. cc @arnaudjnn

**Reply by rashidpathiyil** [reply]
Thank you for your instant response, I am eagerly waiting for the one-click deployment template of Directus

**Reply by rashidpathiyil** [reply]
@WoLfulus made something to leverage the non-matching database information injected by Heroku and he makes it work within Directus. He said it's basically, a custom config loader using CONFIG_PATH env
It would be useful - https://github.com/directus-community/heroku-template

**Reply by wizzuae** [reply]
> Hi @rashidpathiyil, I am the co-founder of Qovery and I guess it's possible to deploy Directus as they support Docker (https://docs.directus.io/guides/installation/docker/). I will be happy to help and see how we can officially support Directus by providing a one-click template deployment in Qovery. cc @arnaudjnn

When can I find the one-click template?

**Reply by arnaudjnn** [reply]
@wizzuae. We should work on it this week. We'll be in touch here soon!

**Reply by wizzuae** [reply]
Is there repo like this?
This is without Dockerfile
 https://github.com/directus-community/heroku-template

**Reply by rashidpathiyil** [reply]
> @wizzuae. We should work on it this week. We'll be in touch here soon!

We eagerly waiting for this.
Thank you @arnaudjnn

**Reply by rashidpathiyil** [reply]
> @wizzuae. We should work on it this week. We'll be in touch here soon!

Any Updates?

**Comment by rijkvanzanten** [comment]
Hi @arnaudjnn, @evoxmusic! 

Lemme know here / [on Discord](https://directus.chat) / through email (rijkvanzanten@me.com) if there's anything you need to know or if there's something we can provide to make this integration smoother 🙂
